src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6618,10 +6620,7 @@ namespace UnitarySystems {
                         this->m_MSHeatingSpeedRatio[i] = 1.0;
                     }
                     if (this->m_HeatingCoilType_Num == HVAC::Coil_HeatingWaterToAirHPVSEquationFit) {
-                        std::string MultispeedType = "UnitarySystemPerformance:Multispeed";
-                        if (this->m_DesignSpecMSHPIndex == -1) {
-                            std::string MultispeedType = "Fan:SystemModel";
-                        }
+                        std::string MultispeedType = (this->m_DesignSpecMSHPIndex == -1) ? "Fan:SystemModel" : "UnitarySystemPerformance:Multispeed";
```

Comment:
[src/EnergyPlus/UnitarySystem.cc:6621]:(style),[variableScope],The scope of the variable 'MultispeedType' can be reduced.
[src/EnergyPlus/UnitarySystem.cc:6623]:(style),[shadowVariable],Local variable 'MultispeedType' shadows outer variable
[src/EnergyPlus/UnitarySystem.cc:6623]:(style),[unreadVariable],Variable 'MultispeedType' is assigned a value that is never used.